packages/ui/src/components/SignIn/SignInFactorOnePasswordCard.tsx

```diff
@@ -64,20 +66,20 @@ export const SignInFactorOnePasswordCard = (props: SignInFactorOnePasswordProps)
   const clerk = useClerk();
 
   const goBack = () => {
-    return navigate('../');
+    void navigate('../');
   };
 
-  const handlePasswordSubmit: React.FormEventHandler = async e => {
+  const handlePasswordSubmit: React.FormEventHandler<HTMLFormElement> = e => {
     e.preventDefault();
-    return signIn
+    void signIn
       .attemptFirstFactor({ strategy: 'password', password: passwordControl.value })
       .then(res => {
         switch (res.status) {
           case 'complete':
             return setActive({
               session: res.createdSessionId,
-              navigate: async ({ session }) => {
-                await navigateOnSetActive({ session, redirectUrl: afterSignInUrl });
+              navigate: ({ session }) => {
+                void navigateOnSetActive({ session, redirectUrl: afterSignInUrl });
```

Comment:
@octoper Why was this changed? It seems to be breaking the redirect after the password flow for me (I'm trying the password flow in the Tanstack test template manually). After this gets called, I see a quick navigation to `/`, and then back to `/sign-in` again. I think because this happens because the navigation is no longer awaitable (which setActive does).